packages/Molecules/Wysiwyg/index.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <!-- Keep app-wysiwyg class here for backward compatibility, as some project may use that class for custom style -->
+  <div class="rpl-wysiwyg tide-wysiwyg app-wysiwyg">
+    <!-- Use dynamic component to render string template in runntime -->
+    <component :is="{template:getTemplate}"/>
+  </div>
+</template>
+
+<script>
+import { wysiwygTranspiler } from './wysiwyg-transpiler'
+
+// To make runtime build work, we need to register the component globally
+import Vue from 'vue'
+import RplButton from '@dpc-sdp/ripple-button'
+import RplDocumentLink from '@dpc-sdp/ripple-document-link'
+import {RplTextLink} from '@dpc-sdp/ripple-link'
+
+Vue.component('rpl-button', RplButton)
+Vue.component('rpl-document-link', RplDocumentLink)
+Vue.component('rpl-text-link', RplTextLink)
+
+export default {
+  name: 'RplWysiwyg',
+  props: {
+    'html': String,
+    'plugins': { type: Array },
```

Comment:
You pass an array of plugins to rplWysiwyg - the idea being these would come from nuxt-tide